rhc-ose-ansible/roles/openshift-management/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Validate OpenShift Token Provided
+  fail: msg="OpenShift Token Not Provided"
+  failed_when: (openshift_token is undefined) or (openshift_token is none) or (openshift_token|trim == '')
+  tags: always
+- name: Set Facts
+  set_fact: kubeconfig="/tmp/openshift-management-{{ ansible_date_time.epoch }}/config"
```

Comment:
This is a solid method of not interrupting the current login by creating a temporary config file. However, I'm noticing that if this playbook fails for any reason, the handler will not clean up the login config. In this instance you will have a cluster-admin credential in a public /tmp/ maintenance folder for any user to point their oc client at and gain cluster-admin rights. Maybe this should default to somewhere in /root/?